### PR TITLE
[7.0] Fix pep8 super checks

### PR DESCRIPTION
**Pull request by dreispt**
Let's make Mr.Travis happy...

### CODE REVIEW

File: crm_department/__openerp__.py [hunk 1]
```diff
@@ -20,20 +20,18 @@
...
 ''',
-    'author' : 'Camptocamp',
+    'author': 'Camptocamp',
     'website': 'http://camptocamp.com',
-    'depends' : ['crm', 'hr'],
-    'data' : ['crm_view.xml'],
-    'demo_xml' : [],
+    'depends': ['crm', 'hr'],
+    'data': ['crm_view.xml'],
+    'demo_xml': [],
```
Comment (pedrobaeza) [inline crm_department/__openerp__.py]: This is deprecated. Use `demo`.

File: project_issue_department/__openerp__.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
     'depends': [
         'project_issue',
         'project_department',
-        ],
+    ],
     'update_xml': [
```
Comment (pedrobaeza) [inline project_issue_department/__openerp__.py]: Use `data`

**Comment by pedrobaeza** [comment]
The rest LGTM.

**Comment by pedrobaeza** [comment]
Thanks :+1:

File: sale_department/__init__.py [hunk 3]
```diff
@@ -1,34 +1 @@
-# -*- coding: utf-8 -*-
```
Comment (yvaucher) [inline sale_department/__init__.py:1]: Should at least this line stay ?

File: invoice_department/__openerp__.py [hunk 4]
```diff
@@ -20,23 +20,20 @@
...
-    'version' : '1.0',
-    'category' : 'Generic Modules/Sales & Purchases',
-    'description':
-'''
-    Add the department on Invoices as well as the related filter and button in the search form.
-
+    'name': 'Invoices with Department Categorization',
+    'version': '1.0',
+    'category': 'Generic Modules/Sales & Purchases',
+    'description': '''\
```
Comment (yvaucher) [inline invoice_department/__openerp__.py]: No need for this \

Comment (dreispt) [inline invoice_department/__openerp__.py]: It avoids a newline before the text.

File: sale_department/__init__.py (hunk 3 above)
Comment (dreispt) [inline sale_department/__init__.py:1]: Just imports; only valid identifiers and no literals expected.
I don't see harm on it, but also don't see any use for it.

File: invoice_department/__openerp__.py (hunk 4 above)
Comment (yvaucher) [inline invoice_department/__openerp__.py]: This is not consistent with `crm_department/__openerp__.py`

Aren't line break ignored in rst ?

File: sale_department/__init__.py (hunk 3 above)
Comment (yvaucher) [inline sale_department/__init__.py:1]: I can live without it so ok. I was just thinking of a time we would add some comments in a **init**.py.

File: invoice_department/__openerp__.py (hunk 4 above)
Comment (dreispt) [inline invoice_department/__openerp__.py]: I don't know. Even if it is, I don't see the need for the extra line break.

Comment (yvaucher) [inline invoice_department/__openerp__.py]: Just try it: http://rst.ninjs.org/ :smiley: 

This \ is not needed

**Comment by dreispt** [comment]
Fixed and removed additional redundancies.

**Comment by yvaucher** [comment]
Thanks :+1:

**Pull request merged**